site/_docs/core-java.md

```diff
@@ -233,14 +233,15 @@ VectorizedRowBatch batch = schema.createRowBatch();
 LongColumnVector x = (LongColumnVector) batch.cols[0];
 LongColumnVector y = (LongColumnVector) batch.cols[1];
 for(int r=0; r < 10000; ++r) {
-  int row = batch.size++;
+  int row = batch.size;
   x.vector[row] = r;
```

Comment:
batch.size must be incremented before the call to writer.addRowBatch on line 241. This change would break the example.